src/tests/integration/mapping.test.ts

```diff
@@ -42,4 +40,36 @@ describe('/api', () =>{
       expect(data.fullDomain).toEqual(`${subDomain}.${domain}`)
     })
   })
+
+  it('Delete mapping', async()=>{
+    const subDomain = `delete${uuidv4()}`
+    const domain = 'albertow'
+    const port = '4500'
+    await fetch(`${apiURL}/api/mappings`, {
+      method: 'POST', 
+      headers: {
+        authorization: ADMIN,
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        domain,
+        subDomain,
+        port,
+      })
+    }).then(r => r.json()).then(async (data)=>{
+      await fetch(`${apiURL}/api/mappings/delete/${data.id}`, {
```

Comment:
path is wrong. should be `${apiURL}/api/mappings/${data.id}`. Is this a bug with the codebase?

Reply:
yes, that is the end point on the server

Reply:
![del](https://user-images.githubusercontent.com/29881336/66436165-4a968980-e9db-11e9-8e91-cf45f00c1cf0.PNG)

Reply:
Will refactor but nearly every other end point has the same issue. I will post an issue to refactor.

Reply:
https://github.com/garageScript/myProxy/pull/246/files